app/src/main/res/layout/fragment_tabhome.xml

```diff
@@ -1,18 +1,191 @@
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        >
+
+        <ImageView
+            android:id="@+id/bannerImageView"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/dp_145"
+            android:scaleType="fitXY"
+            android:contentDescription="@string/banner"
+            android:src="@drawable/gift_banner"
+            />
+
+        <View
+            android:id="@+id/view"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/dp_1"
+            android:layout_below="@+id/bannerImageView"
+            android:layout_margin="@dimen/dp_30"
+            android:background="@color/colorView"
+            />
+
+        <RelativeLayout
+            android:id="@+id/containerNowPlaying"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/view"
+            android:gravity="center_vertical"
+            android:paddingLeft="@dimen/dp_15"
+            android:paddingRight="@dimen/dp_15"
+            >
+            <TextView
+                android:id="@+id/nowPlayingTextView"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/now_playing"
+                android:textSize="@dimen/sp_20"
+                android:textStyle="bold"
+                android:textColor="@color/colorWhite"
+                android:layout_centerVertical="true"
+                android:layout_alignParentLeft="true"
+                />
+            <Button
+                android:id="@+id/moreNowPlayingButton"
+                android:layout_width="@dimen/dp_60"
+                android:layout_height="@dimen/dp_30"
+                android:background="@color/colorOrange"
+                android:text="@string/more"
+                android:textColor="@color/colorWhite"
+                android:textStyle="bold"
+                android:layout_alignParentRight="true"
+                />
+        </RelativeLayout>
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/nowPlayingRecyclerView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/containerNowPlaying"
+            />
+
+        <RelativeLayout
+            android:id="@+id/containerUpComing"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/nowPlayingRecyclerView"
+            android:gravity="center_vertical"
+            android:paddingLeft="@dimen/dp_15"
+            android:paddingRight="@dimen/dp_15"
+            android:layout_marginTop="10dp"
+            >
+            <TextView
```

Comment:
WARNING: [RelativeOverlap] `@id/upComingTextView` can overlap `@id/moreUpComingButton` if @string/upcoming grows due to localized text expansion
 If relative layout has text or button items aligned to left and right sides they can overlap each other due to localized text expansion unless they have mutual constraints like `toEndOf`/`toStartOf`.